src/lane_follow/package.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0"?>
+<package>
+  <name>lane_follow</name>
+  <version>0.0.0</version>
+  <description>Branch dedicated to detecting lanes using the Hough Transform and staying in lane accordingly</description>
+
+  <!-- One maintainer tag required, multiple allowed, one person per tag --> 
+  <maintainer email="leri.ratu@gmail.com">Valerian</maintainer>
+
+
+  <!-- One license tag required, multiple allowed, one license per tag -->
+  <!-- Commonly used license strings: -->
+  <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
+  <license>TODO</license>
+
+  <!-- The *_depend tags are used to specify dependencies -->
+  <!-- Dependencies can be catkin packages or system dependencies -->
+  <buildtool_depend>catkin</buildtool_depend>
+  <build_depend>roscpp</build_depend>
+  <build_depend>rospy</build_depend>
+  <build_depend>sb_utils</build_depend>
+  <build_depend>cv_bridge</build_depend>
+  <run_depend>roscpp</run_depend>
+  <run_depend>rospy</run_depend>
+  <run_depend>cv_bridge</run_depend>
+  <run_depend>sb_utils</run_depend>

```

Comment:
The `sb_utils` `<run_depend>` and `<build_depend>` are currently making CI fail. This is because the `IARRC` repo does not have the `sb_utils` library. I've ported it over, and opened a pull request on this branch (you can open a PR from any branch to any other branch) [here](https://github.com/UBC-Snowbots/IARRC-2017/pull/9). Feel free to merge it (with the `Merge Pull Request` button), and then pull your branch (note that CI will still fail because we're missing the `.cpp` file, but other comments here should fix that), and you should get my changes.